microservices/nris_api/backend/app/__init__.py

```diff
@@ -51,3 +56,10 @@ def register_extensions(app):
     Compress(app)
 
     return None
+
+
+def register_scheduled_jobs(app):
+    if app.config['ENVIRONMENT_NAME'] in ['test', 'prod']:
```

Comment:
I remember SonarQube complaining about these two nested Ifs as a code smell, maybe combine the two ifs?

Reply:
Combined